webdriver-spec.html

```diff
@@ -636,7 +639,7 @@
  <li><p>Remove each entry in <var>endpoints</var>
   for which the concatenation of the <a>url prefix</a>
   and the production in the <i>url</i> column does not match <a>url</a>
-  according to the rules in [[!URI-Template]].
+  according to the rules in [[URI-TEMPLATE]].
```

Comment:
In retrospect I think probably most, if not all, of the `[[…]]` styled references we use should have a `!` and be normative.

I can fix this instance and will file a separate PR for the rest.